Gems/ROS2/Code/Source/RobotImporter/Utils/SourceAssetsStorage.h

```diff
@@ -119,6 +119,7 @@ namespace ROS2::Utils
     UrdfAssetMap CopyAssetForURDFAndCreateAssetMap(
         const AZStd::unordered_set<AZStd::string>& meshesFilenames,
         const AZStd::string& urdfFilename,
+        AZ::Crc32 params_crc,
```

Comment:
These parameters need to be documented. I do not like it CRC32. Please consider adding an optional parameter that extends/overrides the target directory name.

Reply:
changed to optional parameter OutputDirSuffx of type AZStd::string